### PR TITLE
show customised logo

### LINKED ISSUE #334: Show customised logo
__Related issue(s)__

_Add any issues here that may be dependent or must be completed before this issue can be tackled_

__Wireframe link__

_Add link to design here_

---

### Acceptance Criteria:

_REMEMBER THAT WHOEVER WORKS ON THIS ISSUE MUST TICK OFF ALL THE POINTS IN THIS LIST UNLESS THERE IS CLEAR AGREEMENT IN THE COMMENTS TO SAY OTHERWISE. **DO NOT REVIEW A PR INVOLVING THIS ISSUE UNLESS THIS HAS BEEN DONE**_ 

- [ ] Add each acceptance criteria as checklist like this...

### PR DESCRIPTION
**Pull request by RamyAlshurafa**
#### Issues this PR relates to
Close #334

=======

_NOTE: REMEMBER NOT TO REVIEW UNLESS ALL ACCEPTANCE CRITERIA HAS BEEN TICKED OFF_

---

#### Any info the reviewer needs to know to test 
e.g. Add env vars, run npm install

---

#### Any info the team needs to consider before merged into development
e.g. Will a database need to be rebuilt (and is this ok if existing data), add env vars

---

#### Quick Pull Request Checklist

Please make sure you ensure the following has been done before marking your PR as awaiting-review. For full checklist and process review, please refer to Process Guide (#8)

- Branch is up to date with develop
- Project runs
- You can download and install locally, and run the project locally
- All existing and new features run without bugs
- Checked that features/design works across all agreed browsers + devices
- The new work has been checked with Axe accessibility checker and doesn't throw significant errors  
- Linter is used
- Code is consistent with agreed team style
- Naming of files is consistent with project 
- Tests all pass
- No console logs

### CODE REVIEW

**Review by thejoefriel — CHANGES_REQUESTED** [review]
- [x] When I go to the sign up page as a therapist, it still shows me the Duet logo rather than the organisation's logo: 
image

- [ ] When I go to /chiltern it still throws a 404. Can we have this take you to /chiltern/log-in and show the logo please?

- [x] When I go to sign up as a client for a therapist I should see the organisation logo instead of Duet 
image

File: client/src/components/CustomLogo/index.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+export const StyledImage = styled(Image)`
+  max-width: 120px;
+
+  ${({ theme }) => theme.media.mobile} {
+    max-width: 90px;
+  }
+`;
+export const Logo = styled.img`
+  max-width: 120px;
```
Comment (thejoefriel) [inline client/src/components/CustomLogo/index.js:12]: can we add max-height: 90px;

File: client/src/components/CustomLogo/index.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+  ${({ theme }) => theme.media.mobile} {
+    max-width: 90px;
```
Comment (thejoefriel) [inline client/src/components/CustomLogo/index.js:15]: can we add max-height: 70px;

**Review by thejoefriel — CHANGES_REQUESTED** [review]

File: client/src/components/CustomLogo/index.js (hunk 1 above)
Comment (RamyAlshurafa) [reply client/src/components/CustomLogo/index.js:12]: done

File: client/src/components/CustomLogo/index.js (hunk 2 above)
Comment (RamyAlshurafa) [reply client/src/components/CustomLogo/index.js:15]: done

**Review by thejoefriel — CHANGES_REQUESTED** [review]
- [x] Remove the "inviteToken=" bit
image

- [x] Welcome pages when signing up (as therapist or client) need to say the organisation name rather than Chiltern Music Therapy. Screenshot below
- [x] Logo is squashed in terms of height on these welcome pages. Screen shot below
image

File: client/src/components/CustomLogo/index.js (hunk 2 above)
Comment (RamyAlshurafa) [reply client/src/components/CustomLogo/index.js:16]: done

File: client/src/components/CustomLogo/index.js (hunk 1 above)
Comment (RamyAlshurafa) [reply client/src/components/CustomLogo/index.js:12]: done

**Review by thejoefriel — APPROVED** [review]

**Pull request merged**